napari_pyclesperanto_assistant/_tests/test_workflow_processing_labels.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from napari import Viewer
+
+@pytest.fixture
+def viewer(qtbot):
+    return  Viewer(show=False)
+
+def test_workflow_processing_labels(viewer):
```

Comment:
until we have a ton of tests, let's just keep them in a single file for now.  You can rename "test_dock_widget.py" to something more general and keep the test names specific.  Then you can move this there, reuse the viewer fixture there and combine as follows (also, I should have yielded and closed the viewer anyway, as shown below)

```python
@pytest.fixture
def viewer(qtbot):
    viewer = Viewer(show=False)
    yield viewer
    viewer.close()

@pytest.fixture
def assistant(viewer):
    viewer.window.add_plugin_dock_widget("clEsperanto", "Assistant")
    dw = viewer.window._dock_widgets["clEsperanto: Assistant"]
    return dw.widget()
```